server/pulp/server/event/http.py

```diff
@@ -25,56 +24,36 @@
 def handle_event(notifier_config, event):
     # fire the actual http push function off in a separate thread to keep
     # pulp from blocking or deadlocking due to the tasking subsystem
-
     data = event.data()
-
     _logger.info(data)
-
-    body = json.dumps(data, default=json_util.default)
```

Comment:
What is the reason to remove this line? I'm not particularly familiar with Pulp's json_util module.